### PR TITLE
test change

### PR DESCRIPTION
**Pull request by silvershotm1**

### CODE REVIEW

**Comment by jacoby22** [comment]
Driver: Michael B.
Navigator: Nick J.
- We spent roughly 2 hours working on this assignment. Most time was spent working through the Git Workflow and figuring out the JQuery .find() and .attr() methods.

**Comment by silvershotm1** [comment]
Driver: Michael Bishop
Navigator: Nick Jacoby

Spent good amount of time with Github and that frustration. We did go through all the TODO and what needed to be done first. After making an outline we came tackled the problem. Did some console log and dev tools for some debugging. At the end things seemed to come together.

File: class-01-jquery-and-the-dom/pair-assignment/starter-code/scripts/article.js [hunk 1]
```diff
@@ -4,39 +4,50 @@ function Article (opts) {
...
+  this.category = opts.category;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
+
 }
 
 Article.prototype.toHtml = function() {
   var $newArticle = $('article.template').clone();
 
-  $newArticle.attr('data-category', this.category);
```
Comment (tiodan81) [inline class-01-jquery-and-the-dom/pair-assignment/starter-code/scripts/article.js]: make sure you don't leave the category out

File: class-01-jquery-and-the-dom/pair-assignment/starter-code/scripts/article.js [hunk 2]
```diff
@@ -4,39 +4,50 @@ function Article (opts) {
...
 rawData.forEach(function(ele) {
   articles.push(new Article(ele));
-})
+});
 
 articles.forEach(function(a){
-  $('#articles').append(a.toHtml())
+  $('#articles').append(a.toHtml());
 });
+console.log(articles[0]);
```
Comment (tiodan81) [inline class-01-jquery-and-the-dom/pair-assignment/starter-code/scripts/article.js]: remove test commands like these before you submit the assignment

**Pull request closed without merging**